config.example/group_vars/all.yml

```diff
@@ -8,6 +8,9 @@ ssh_password_authentication: 'yes'
 # If this is `false`, pass the `-K` flag to all Ansible commands to prompt for the SUDO password
 enable_sudo_without_password: true
 
+# Configure the default ansible user
+#ansible_user: ubuntu
```

Comment:
See https://github.com/NVIDIA/deepops/pull/107

Reply:
Ack. I'll rebase this branch after #107 lands and fix the `sed` command to refer to inventory.

Reply:
Done

Reply:
@ajdecon - I don't think we need it. Should take out the sed command since all ansible-playbook commands explicitly have `-e "ansible_user=vagrant"` set. Should retest without the sed modification.

Reply:
Confirmed that simply removing the `sed` command works fine, went ahead and made that change. Removed addition to `config.example/group_vars/all.yml`.